VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -35,28 +38,28 @@
 public class CourseDiscussionPostsThreadFragment extends CourseDiscussionPostsBaseFragment {
 
     @InjectView(R.id.discussion_posts_filter_layout)
-    RelativeLayout discussionPostsFilterLayout;
+    private RelativeLayout discussionPostsFilterLayout;
 
     @InjectView(R.id.discussion_posts_filter_text_view)
-    TextView discussionPostsFilterTextView;
+    private TextView discussionPostsFilterTextView;
 
     @InjectView(R.id.discussion_posts_sort_layout)
-    RelativeLayout discussionPostsSortLayout;
+    private RelativeLayout discussionPostsSortLayout;
 
     @InjectView(R.id.discussion_posts_sort_text_view)
-    TextView discussionPostsSortTextView;
+    private TextView discussionPostsSortTextView;
 
     @InjectView(R.id.create_new_item_text_view)
-    TextView createNewPostTextView;
+    private TextView createNewPostTextView;
 
     @InjectView(R.id.create_new_item_layout)
-    ViewGroup createNewPostLayout;
+    private ViewGroup createNewPostLayout;
```

Comment:
You can set the fields in `CourseDiscussionResponsesFragment` to private as well I guess.